watertap/flowsheets/dye_desalination/dye_desalination.py

```diff
@@ -627,6 +635,13 @@ def add_costing(m):
 
     m.fs.zo_costing = ZeroOrderCosting(case_study_definition=source_file)
     m.fs.ro_costing = WaterTAPCosting()
+    m.fs.ro_costing.electricity_cost = value(m.fs.zo_costing.electricity_cost)
+    m.fs.ro_costing.base_currency = pyunits.USD_2020
+    m.fs.ro_costing.utilization_factor.fix(0.85)
+    # Assume the same capital recovery factor as zo_costing
+    m.fs.ro_costing.capital_recovery_factor.fix(0.065051435)
```

Comment:
@MarcusHolly any reason why we don't just point to the variable itself to get the value instead of hard-coding here?

Reply:
Nope - I'll update this